From 3f2a2973b23fa86c810aab49eeb8ff5f97a02720 Mon Sep 17 00:00:00 2001
From: Yasunori Goto <y-goto@fujitsu.com>
Date: Wed, 3 Feb 2021 19:17:07 +0900
Subject: [PATCH] Documentation/ndctl: fix self-reference of ndctl
 disable-namespace

Upstream: v73
Git-commit: 3f2a2973b23fa86c810aab49eeb8ff5f97a02720

The man manual of ndctl disable-namespace link to itself at See
Also section. It should be enable-namespace instead of it.

Signed-off-by: Yasunori Goto <y-goto@fujitsu.com>
Signed-off-by: Vishal Verma <vishal.l.verma@intel.com>
Link: https://github.com/pmem/ndctl/pull/160
---
 Documentation/ndctl/ndctl-disable-namespace.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/ndctl/ndctl-disable-namespace.txt b/Documentation/ndctl/ndctl-disable-namespace.txt
index 5d3a8be..187348f 100644
--- a/Documentation/ndctl/ndctl-disable-namespace.txt
+++ b/Documentation/ndctl/ndctl-disable-namespace.txt
@@ -22,4 +22,4 @@ include::../copyright.txt[]
 
 SEE ALSO
 --------
-linkndctl:ndctl-disable-namespace[1]
+linkndctl:ndctl-enable-namespace[1]
-- 
2.35.1

